pre-registration/pre-registration-core/src/main/java/io/mosip/preregistration/core/util/NotificationUtil.java

```diff
@@ -2,7 +2,9 @@
 
 import java.io.IOException;
 import java.util.Date;
+import java.util.Optional;

```

Comment:
@MeghaUttamTanga Can you please remove the additional imports ?

Reply:
removed the extra imports and pushed the code